dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/model/dataRequest/SingleDataRequestResponse.kt

```diff
@@ -0,0 +1,21 @@
+package org.dataland.datalandcommunitymanager.model.dataRequest
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+/**
+ * --- API model ---
+ * Contains all relevant info that a user should receive as response when posting a bulk data request to Dataland.
+ * @param message contains a text about the result of Dataland processing the bulk data request
+ * @param reportingPeriodsOfStoredDataRequests reporting periods corresponding to the accepted and stored data requests
+ * @param reportingPeriodsOfDuplicateDataRequests reporting periods corresponding to duplicate data requests on Dataland
+ */
+data class SingleDataRequestResponse(
+    @field:JsonProperty(required = true)
+    val message: String,
+
+    @field:JsonProperty(required = true)
+    val reportingPeriodsOfStoredDataRequests: List<String>,
```

Comment:
I would rather call it reportingPeriodsOfAcceptedDataRequests. Stored is not so clear, as it can mean the reporting periods that have already been stored, aka duplicates